src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
+    blue,
+    yellow,
+    black,
+    white

```

Comment:
Please see [this article](https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes) to learn how to name ENUMs